src/morphocut/annotation.py

```diff
@@ -0,0 +1,88 @@
+
+"""
+Operations for annotating images.
+"""
+import numpy as np
+from skimage import img_as_ubyte, img_as_float
+from skimage.morphology import binary_dilation, disk
+import skimage.exposure
+import skimage.io
+import skimage.measure
+import skimage.segmentation
+
+from morphocut import Node, Output, RawOrVariable, ReturnOutputs
+
+@ReturnOutputs
+@Output("contour")
+class DrawContours(Node):
+
+    def __init__(self, image, mask, dilate_rel=0, dilate_abs=0):
+        super().__init__()
+        self.image = image
+        self.mask = mask
+        self.dilate_rel = dilate_rel
+        self.dilate_abs = dilate_abs
+
+    def transform(self, image, mask):
+        # Calculate radius of the dilation
+        area = np.sum(mask)
+        radius = self.dilate_rel * np.sqrt(area) + self.dilate_abs
+
+        # Dilate mask
+        dilated_mask = binary_dilation(
+            mask,
+            disk(radius))
+
+        # Draw contours
+        boundary_mask = skimage.segmentation.find_boundaries(dilated_mask, mode="outer")
+
```

Comment:
You need a `skimage.color.gray2rgb` at this point to make sure you're operating on a color image with three channels.